schemas/com.snowplowanalytics.snowplow.ecommerce/transaction/jsonschema/1-0-0

```diff
@@ -0,0 +1,78 @@
+{
+    "$schema": "http://iglucentral.com/schemas/com.snowplowanalytics.self-desc/schema/jsonschema/1-0-0#",
+    "description": "Schema for a transaction entity in Ecommerce",
+    "self": {
+        "vendor": "com.snowplowanalytics.snowplow.ecommerce",
+        "name": "transaction",
+        "format": "jsonschema",
+        "version": "1-0-0"
+    },
+    "type": "object",
+    "properties": {
+        "transaction_id": {
+            "description": "The ID of the transaction.",
+            "type": "string",
+            "maxLength": 4096
+        },
+        "revenue" : {
+            "type": "number",
+            "description": "The revenue of the transaction.",
+            "multipleOf": 0.01,
+            "minimum": 0.00,
+            "maximum": 2147483647
+        },
+        "currency": {
+            "type": "string",
+            "description": "The currency used for the transaction (ISO 4217).",
+            "maxLength": 3,
+            "minLength": 3
+        },
+        "payment_method": {
+            "type": "string",
+            "description": "The payment method used for the transaction.",
+            "maxLength": 99
```

Comment:
In the checkout schema, the max length of payment_method is 128, but here you set it to 99.  Is this deliberate to have a discrepancy?

Reply:
Nope, good catch. Fixing